frontend/js/components/TrainersInBattle.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+const TrainersInBattle = (props) => {
+  const { creator, opponent } = props;
+  return (
+    <span>
+      {creator.name} VS {opponent.name}
+    </span>
+  );
+};
+
+TrainersInBattle.propTypes = {
+  creator: PropTypes.object,
+  opponent: PropTypes.object,
+};
+
+const mapStateToProps = (state, { creator, opponent }) => {
```

Comment:
Use nomenclature `entityId` for ids and `entity` for the actual object.
Example: `creatorId` and `creator`